.nix/polkadot-bin.nix

```diff
@@ -24,5 +29,4 @@ rustPlatform.buildRustPackage rec {
   LIBCLANG_PATH = "${llvmPackages.libclang.lib}/lib";
   PROTOC = "${protobuf}/bin/protoc";
   ROCKSDB_LIB_DIR = "${rocksdb}/lib";
-  meta = { mainProgram = "polkadot"; };
```

Comment:
Please keep the metadata

Reply:
the name is not shared across chains, so i moved it up(L18). so it would be easier to replace all common shared attrs later